modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -1931,7 +1994,7 @@ private String onOff(boolean b) {
     }
 
     /**
-     * @return Whether or not REST is enabled.
+     * @return {@code true} if the rest processor is enabled, {@code false} the otherwise.
```

Comment:
*REST* in capital case seems more common (in a javadoc).

Reply:
fixed